drake/doc/code_review_checklist.rst

```diff
@@ -66,14 +66,16 @@ Are you using pointers well?
 Did you document your API?  Did you comment your complexity?
 ============================================================
 
-- Every nontrivial public symbol must have a doxygen-formatted
+- Every nontrivial public symbol must have a Doxygen-formatted
   comment.
-
 - If you are uncertain of your formatting, consider
-  :ref:`generating the doxygen <documentation-generation-instructions>`
+  :ref:`generating the Doxygen <documentation-generation-instructions>`
   and checking how it looks in a browser.
+- Only use Doxygen comments (``///`` or ``/** */``) on published APIs (public
+  or protected classes and methods).  Code with private access or declared in
+  ``.cc`` files should *not* use the Doxygen format.
```

Comment:
why is this?  doxygen has an option to not publish private methods.  isn't it better to document everything (w/ doxygen) and have it disabled for the public doxygen?